src/github.com/matrix-org/dendrite/roomserver/query/query.go

```diff
@@ -34,7 +34,7 @@ type RoomserverQueryAPIDatabase interface {
 	RoomNID(roomID string) (types.RoomNID, error)
 	// Lookup event references for the latest events in the room and the current state snapshot.
 	// Returns an error if there was a problem talking to the database.
-	LatestEventIDs(roomNID types.RoomNID) ([]gomatrixserverlib.EventReference, types.StateSnapshotNID, error)
+	LatestEventIDs(roomNID types.RoomNID) ([]gomatrixserverlib.EventReference, types.StateSnapshotNID, int64, error)
```

Comment:
Please explain what the `int64` is. I know its depth, but having to look at the implemention rather than the interface to work that out seems somewhat backwards :)